src/table/src/table/scan.rs

```diff
@@ -152,6 +159,28 @@ impl ExecutionPlan for RegionScanExec {
     fn metrics(&self) -> Option<MetricsSet> {
         Some(self.metric.clone_inner())
     }
+
+    fn statistics(&self) -> DfResult<Statistics> {
+        let statistics = if self.append_mode {
+            let column_statistics = self
+                .arrow_schema
+                .fields
+                .iter()
+                .map(|_| ColumnStatistics {
+                    distinct_count: Precision::Exact(self.total_rows),
```

Comment:
The distinct count may not be `total_rows` as each column may have duplicated values.

Reply:
> The distinct count may not be `total_rows` as each column may have duplicated values.

`ColumnStatistics` is only read iff statement is `select count(*) from table` (without predicates) and `append_mode` is true. Maybe I should add some doc here.

Reply:
Maybe we can document it in this method.